infrastructure/kube/keep-prd/bitcoin/bitcoind/bitcoind-data-bitcoind-1-pvc.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: bitcoind-data-bitcoind-1
```

Comment:
I am looking at the list of PVCs in GCP and I see `bitcoind-data-bitcoind-1` and `bitcoind-data-bitcoind-0`. To confirm my understanding: we are running two replicas of bitcoind, each with its own PVC. Then, why don't we have `bitcoind-data-bitcoind-0-pvc.yaml` file here as well?